packages/kbn-shared-ux-services/src/services/editors.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/**
+ * TODO: `DataView` is a class exported by `src/plugins/data_views/public`.  Since this service
+ * is contained in this package-- and packages can only depend on other packages and never on
+ * plugins-- we have to set this to `unknown`.  If and when `DataView` is exported from a
+ * stateless package, we can remove this.
+ */
+type DataView = unknown;
```

Comment:
I accidentally left a comment in the PoC PR, so I'll just leave it here 😅 
https://github.com/elastic/kibana/pull/127419/files#r826252775